.github/workflows/setup.yml

```diff
@@ -0,0 +1,62 @@
+name: Setup
+
+on:
+  workflow_call:
+    inputs:
+      aws-account-id:
+        description: 'AWS Access Account ID'
+        required: false
+        default: '249832953260'
+        type: string
+      aws-region:
+        description: 'AWS Region for ECR'
+        required: false
+        default: ap-southeast-2
+        type: string
+      image-tag:
+        description: 'Image Tag'
+        required: false
+        default: 3832bee3b58efae7fb29e5c8517140300c4c9daf
+        type: string
+      node-version:
+        description: 'Node Version'
+        required: false
+        default: '14.17'
+        type: string
+    outputs:
+      pr-number:
+        description: 'The current PR number'
+        value: ${{ github.event.number || github.event.inputs.prNumber }}
+      integration-image-name:
+        description: 'Name of integration image'
+        value: ${{ jobs.image-names.outputs.integration-image-name }}
+      squidex-image-name:
+        description: 'Name of Squidex image'
+        value: ${{ jobs.image-names.outputs.squidex-image-name }}
+      node-image-name:
+        description: 'Name of node image'
+        value: ${{ jobs.image-names.outputs.node-image-name }}
+jobs:
+  image-names:
```

Comment:
I remember this loading some data by logging into aws. Since this is gone now, can't we simplify this as a shell script?